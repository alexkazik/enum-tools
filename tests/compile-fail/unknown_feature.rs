#![no_implicit_prelude]

use ::enum_tools::EnumTools;

#[derive(EnumTools, Debug, Clone, Copy, Eq, PartialEq)]
#[enum_tools(unknown1)]
#[repr(i8)]
pub(crate) enum EG {
    #[enum_tools(rename = "A*")]
    A,
    B,
    C,
    D,
}

fn main() {}
