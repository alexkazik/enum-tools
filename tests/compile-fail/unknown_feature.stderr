error: unknown feature
 --> tests/compile-fail/unknown_feature.rs:6:14
  |
6 | #[enum_tools(unknown1)]
  |              ^^^^^^^^
